### ISSUE TITLE
CRAN check errors

### ISSUE THREAD

**Issue by goergen95** [mapme-initiative/mapme.biodiversity#227]
Received a message from CRAN:

```
Please see the problems shown on
[<https://cran.r-project.org/web/checks/check_results_mapme.biodiversity.html>](https://cran.r-project.org/web/checks/check_results_mapme.biodiversity.html).

Please correct before 2024-01-11 to safely retain your package on CRAN.

It seems we need to remind you of the CRAN policy:

'Packages which use Internet resources should fail gracefully with an informative message
if the resource is not available or has changed (and not give a check warning nor error).'

This needs correction whether or not the resource recovers.

The CRAN Team
```

**Comment by goergen95** [comment]
This is fixed since merging 59a4c93d. We need a new CRAN release.

**Comment by goergen95** [comment]
For completeness, I currently see on CRAN:

```
  Error(s) in re-building vignettes:
    ...
  --- re-building ‘quickstart.Rmd’ using rmarkdown
  
  Quitting from lines 205-206 [get_chirps] (quickstart.Rmd)
  Error: processing vignette 'quickstart.Rmd' failed with diagnostics:
  Download for resource chirps failed. Returning unmodified portfolio object.
  --- failed re-building ‘quickstart.Rmd’
  
  --- re-building ‘terminology.Rmd’ using rmarkdown
  --- finished re-building ‘terminology.Rmd’
  
  SUMMARY: processing the following file failed:
    ‘quickstart.Rmd’
```
